guides/source/form_helpers.md

```diff
@@ -809,16 +889,22 @@ The form builder used also determines what happens when you do:
 
 If `f` is an instance of `ActionView::Helpers::FormBuilder`, then this will render the `form` partial, setting the partial's object to the form builder. If the form builder is of class `LabellingFormBuilder`, then the `labelling_form` partial would be rendered instead.
 
-Understanding Parameter Naming Conventions
-------------------------------------------
+The form builder customizations do add indirection (and may seem like an overkill for the simple example above). Choose between different customizations, extending `FormBuilder` class or creating helpers, based on how frequently your forms use the custom elements.
```

Comment:
indirection = when you look at code like `labeled_form_with model: @person do |form|` you can't tell what it does, unless you do find "labeled_form_with" is defined and read its code.

Reply:
I also had this question. Since most readers won't have this comment to look back on for clarification, is there a way that we can make this more clear in the text itself?

Reply:
Went with "Form builder customizations, such as `LabellingFormBuilder`, do hide the implementation details". Open to other wording if someone thinks of something better.